Next.js <> OpenSea-js 7.1.0 build error

### Component

API

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

7.1.0

### What function is the bug in?

_No response_

### Operating System

macOS (Apple Silicon)

### Describe the bug

 After upgrading from `opensea-js 7.0.9` I'm getting the following error when building : 
 ```
 ✓ Linting and checking validity of types
   Creating an optimized production build ...
Failed to compile.

../node_modules/opensea-js/lib/api/api.js
Module not found: Can't resolve 'timers/promises'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../node_modules/opensea-js/lib/sdk.js
../node_modules/opensea-js/lib/index.js
```
I'm using `next.js v13.4.1` and `node v20.9.0`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "opensea-js",
-  "version": "7.1.0",
+  "version": "7.1.1",
   "description": "TypeScript SDK for the OpenSea marketplace helps developers build new experiences using NFTs and our marketplace data",
   "license": "MIT",
   "author": "OpenSea Developers",

--- a/src/api/api.ts
+++ b/src/api/api.ts
@@ -1,4 +1,3 @@
-import { setTimeout } from "timers/promises";
 import { ethers } from "ethers";
 import {
   getCollectionPath,
@@ -67,6 +66,14 @@ import {
   isTestChain,
   accountFromJSON,
 } from "../utils/utils";
+
+function stall(duration: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, duration);
+  });
+}
 
 /**
  * The API class for the OpenSea SDK.
@@ -633,7 +640,7 @@ export class OpenSeaAPI {
         `Fetch attempt ${attempt} failed with status ${resp.statusCode}`,
       );
       // Wait 1s between tries
-      await setTimeout(1000);
+      await stall(1000);
       return true;
     };
 

--- a/test/integration/postOrder.spec.ts
+++ b/test/integration/postOrder.spec.ts
@@ -123,7 +123,7 @@ suite("SDK: order posting", () => {
     const postOrderRequest = {
       collectionSlug: collection.collection,
       accountAddress: walletAddress,
-      amount: OFFER_AMOUNT,
+      amount: 0.0001,
       quantity: 1,
       paymentTokenAddress,
     };

